src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -594,31 +594,34 @@ export default class AuthorizationService {
     cars.metadata = authorizationFilter.metadata;
     // Add Authorizations
     cars.canCreate = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.CAR, Action.CREATE, authorizationFilter);
+    cars.canListUsers = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.USER, Action.LIST, authorizationFilter);
+    cars.canListCarCatalog = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.CAR_CATALOG, Action.LIST, authorizationFilter);
+    cars.canCreatePoolCar = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.CAR, Action.CREATE_POOL_CAR, authorizationFilter);
     for (const car of cars.result) {
       await AuthorizationService.addCarAuthorizations(tenant, userToken, car, authorizationFilter);
     }
   }
 
   public static async addCarAuthorizations(tenant: Tenant, userToken: UserToken, car: Car, authorizationFilter: AuthorizationFilter): Promise<void> {
     car.canRead = true; // Always true as it should be filtered upfront
-    car.canDelete = await AuthorizationService.canPerformAuthorizationAction(
-      tenant, userToken, Entity.CAR, Action.DELETE, authorizationFilter, { CarID: car.id }, car);
-    car.canUpdate = await AuthorizationService.canPerformAuthorizationAction(
-      tenant, userToken, Entity.CAR, Action.UPDATE, authorizationFilter, { CarID: car.id }, car);
+    car.canDelete = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.CAR, Action.DELETE, authorizationFilter, { CarID: car.id }, car);
+    car.canUpdate = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.CAR, Action.UPDATE, authorizationFilter, { CarID: car.id }, car);
+    car.canListUsers = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.USER, Action.LIST, authorizationFilter);
+    car.canCreatePoolCar = await AuthorizationService.canPerformAuthorizationAction(tenant, userToken, Entity.CAR, Action.CREATE_POOL_CAR, authorizationFilter);
```

Comment:
Too specific.

Reply:
There was a rule in the GUI that only allowed admin to create a pool car (site admin not allowed to see the pool car option in the create dialog) maybe this is not the desired behavior ? 
Actually a site admin can edit a pool car but is not allowed to transform a "normal car" to a "pool car" but only the other way round: pool car -> normal car

Reply:
Create pop up for site admin: no pool car option (same as edit dialog for pool car)
![image](https://user-images.githubusercontent.com/1226644/151849311-755a20aa-8f4e-4e93-9646-20a2a5c8b761.png)
Site admin edit pool car
![image](https://user-images.githubusercontent.com/1226644/151850305-4ae071db-1139-4acb-8d42-d3395ed32a35.png)

Reply:
Normally, only Admin/Site Admin can create a pool car and change it to another type of car and vice-versa.
So I would expect in the UI that for Basic user you don't display the pool car type (with metaData?) and in the backend an assertion that checks that basic user cannot set this value.
Let me know.